--- Socket6-0.23/Socket6.pm.orig	2008-11-01 20:08:39.000000000 +0100
+++ Socket6-0.23/Socket6.pm	2013-06-13 13:26:12.387269206 +0200
@@ -78,6 +78,8 @@
 
 Functions supplied are:
 
+=over
+
 =item inet_pton FAMILY, TEXT_ADDRESS
 
     This function takes an IP address in presentation (or string) format
@@ -204,8 +206,6 @@
 
     This function returns the 16-octet loopback address.
 
-=over
-
 =back
 
 =cut
